tox.ini

```diff
@@ -25,8 +25,7 @@ deps =
     mypy: {[_mypy]deps}
 passenv = INTEGRATION_TEST
 commands =
-    test: python -bb -m pytest -rs {posargs}
```

Comment:
The `-rs` is now handled by the options in `setup.cfg`, right?

Reply:
Indeed. `-rs` only selected to show the SKIPPED tests but ignored the rest. `-ra` shows everything except passed
![image](https://github.com/beetbox/beets/assets/16212750/ab6b73e9-cc7c-4aad-8bb9-0dacafb41cd5)